tests/db_engine_specs/hive_tests.py

```diff
@@ -209,7 +209,7 @@ def test_create_table_from_csv_if_exists_fail_with_schema(mock_table, mock_g):
 
 @mock.patch(
     "superset.db_engine_specs.hive.config",
-    {**app.config, "CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC": lambda *args: True},
+    {**app.config, "CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC": lambda *args: ""},
```

Comment:
The `CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC` function returns a `str` and not a `boolean`.